hw/ip/aes/dv/test_vectors/read_nist_vectors.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python
+import os
+import codecs
+import sys
+import argparse
+
+LOCAL_PROJ = os.getenv('LOCAL_PROJ')
+
+
+#### CREATE HELP AND MAKE SCRIPT ACCEPT COMMANDLINE INPUTS ########
+
+#num_arg = len(sys.argv)
+#print "arguments" + str(sys.argv)
+#print "num arg" + " " +str(num_arg)
+#
+#parser = argparse.ArgumentParser(description='Convert NIST test Vector file into rsp for OpenTitan')
+#parser.add_argument('-i', type=str, help='input NIST file')
+#args = parser.parse_args()
+
+
+
+#open file
+
+read_file  = open (LOCAL_PROJ + "/../../opentitan/golden/nist_vectors.txt", "r")
```

Comment:
is this `nist_vectors.txt` the downloaded (and renamed) vector file from NIST website?

Reply:
no it is a converted pdf, you can download it from the link in the file (also mentioned in the README)
do you want me to add the txt file also? might be helpful - but I obviously can add a license header to that file. 
so CI would probably fail

Reply:
No need to add that file as of now, it's fine to accept the "hardcoded" `nist_vectors_pkg.sv` that you've added as we've essentially hardcoded all of the other NIST vectors.
We'll take care of this at a later time.